how-to/use-lp-shell.rst

```diff
@@ -0,0 +1,223 @@
+=====================
+How to Use lp-shell
+=====================
+
+lp-shell is a CLI tool in the form of an interactive Python shell that
+provides API functionality over Launchpad for the developers and users in a
+quick and easy way. It is especially useful for providing support to users,
+as well as to automate and fasten various Launchpad tasks and it is practically
+necessary for some features that do not provide a UI alternative, especially
+when they are early in their life-cycle.
```

Comment:
While this is kinda hinting at that, can we mention explicitly that `lp-shell` provides an interactive Python API client for the Launchpad API?